test/user/user.e2e-spec.ts

```diff
@@ -30,39 +30,14 @@ describe('UserController (e2e)', () => {
     app.useGlobalPipes(new ValidationPipe());
     await app.init();
 
-    // Create a User
-    const registerInput: RegisterInput = {
-      name: 'e2etester',
-      username: 'e2etester',
-      password: '12345678',
-      roles: [ROLE.USER],
-      isAccountDisabled: false,
-      email: 'e2etester@random.com',
-    };
-
-    const loginInput: LoginInput = {
-      username: 'e2etester',
-      password: '12345678',
-    };
-
-    await request(app.getHttpServer())
-      .post('/auth/register')
-      .send(registerInput)
-      .expect(HttpStatus.CREATED);
-
-    const tokenResponse = await request(app.getHttpServer())
-      .post('/auth/login')
-      .send(loginInput)
-      .expect(HttpStatus.OK);
-
-    accessToken = tokenResponse.body.data.accessToken;
+    authTokenForAdmin = await createAdminUser(app);
```

Comment:
@sivanpayyadakath 
How about the function returning `{ adminUser, authTokenForAdmin }`

Do you think the returned `adminUser` would be useful?

Reply:
@yashmurty 
Yeah. I would be a nice idea. 👍